tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
I may need you to break down what you mean by infrastructure and fixtures here.  For example, would the general caseHolder for a package be `gpii.test.package.caseHolder` and the one specific to a test be `gpii.tests.package.specificThingUnderTest.caseHolder`?  Please clarify.